eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -25,6 +23,19 @@
 from eox_core.edxapp_wrapper.coursekey import get_valid_course_key, validate_org
 from eox_core.edxapp_wrapper.users import check_edxapp_account_conflicts
 
+try:
```

Comment:
Imo we should do this as a new backend `eox_core/edxapp_wrapper/backends/enrollment_j_v1.py`

Reply:
even considering that we are only fixing the import?

Reply:
Since the change is backwards compatible with hawhtorn I would say that changing the _h_v1 backend is good enough